src/js/Features.js

```diff
@@ -165,23 +165,17 @@ Features.prototype.generateElements = function (featuresElements) {
             if (!self._features[i].hideName) {
                 featureName = `<td><div>${self._features[i].name}</div></td>`;
             }
+            const newElement = $(`<tr><td><input class="feature toggle" id="feature-${i}" \
+                " name="${self._features[i].name}" title="${self._features[i].name}" type= \
+                "checkbox"/></td><td><div>${featureName} \
+                </div><span class="xs" i18n="feature${self._features[i].name}"></span></td> \
+                <td><span class="sm-min" i18n="feature${self._features[i].name}"></span>' \
+                ${feature_tip_html}</td></tr>`);
```

Comment:
I think that the back slash `\` is not needed. The backtick ` marks the use of "template" literals and they let the use of multiline directly: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Multi-line_strings